Sources/StreamDeckKit/Session/InternalStreamDeckSession.swift

```diff
@@ -99,6 +103,9 @@ final actor InternalStreamDeckSession {
 
             guard ret == kIOReturnSuccess else {
                 os_log(.error, "Failed opening service with error: \(String(ioReturn: ret)).")
+                if ret == IOErrorCode.notPermitted.rawValue {
```

Comment:
`if ret == kIOReturnNotPermitted`

Reply:
Boah ich könnte schwören autocomplete hat mir das nicht gezeigt!

Reply:
I added `#include <IOKit/IOReturn.h>` to StreamDeckCApi.h. But without success.

Reply:
I think it is because this value is just a `#define`.
Try to declare `extern const IOReturn sdkIOReturnNotPermitted;` in StreamDeckCApi.h and then add `const IOReturn sdkIOReturnNotPermitted = kIOReturnNotPermitted;` to StreamDEckCApi.c.

Reply:
@beyama I updated according to your hint.